contracts/extension/plugin/PluginRegistry.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+import "../../lib/TWStringSet.sol";
+import "../interface/plugin/IPluginRegistry.sol";
+import "../PermissionsEnumerable.sol";
+import "./PluginData.sol";
+
+contract PluginRegistry is IPluginRegistry, PermissionsEnumerable, PluginData {
```

Comment:
its interesting that you're extending PluginData even though its built with the plugin pattern internally. That means that we could rewrite our ContractKit extensions in the plugin pattern and use them both as plugins or as extended classes right?